doc/install.rst

```diff
@@ -0,0 +1,83 @@
+Build Instructions
+==================
+
+Dependencies
+------------
+
+We are using colcon_ as build tool and typically use `ros2 run` to execute our
+applications.  While we are not really depending on any ROS_ packages, this
+means a basic ROS 2 installation is recommended.
+
+We are testing on Ubuntu 20.04 with ROS Foxy.  Other versions may work as well
+but are not officially supported.
+
```

Comment:
We should probably mention actual build dependencies as well here but on the other hand, it can easily get out of hand to maintain dependency lists in the docs of all packages...

Reply:
I don't think we should mention this. The entire point about having the treep setup is that it hides the dependencies. If someone wants to understand the dependencies, they can look at the CMakeLists.txt file.

Reply:
I was thinking about system dependencies (Eigen, Boost, etc.).  I think listing them all here is to hard to maintain, maybe we could link to a singularity or docker container that contains all that is needed (this is what I do for the TriFinger project).

Reply:
Oh, good point. Yes, having a link to a docker script would be great!